server/backend/src/cameras/cameras.service.spec.ts

```diff
@@ -16,7 +15,9 @@ describe('CamerasService', () => {
   beforeEach(async () => {
     mockedNotificationRepository = {
       notifications: [Notification],
-      persist(notification: Notification) {},
+      persist: (notification: Notification) => {
```

Comment:
quick question: what does persist do?

Reply:
`persist` tells a MikroORM repository to manage an entity so it'll push changes to that object to the database. Their [docs](https://mikro-orm.io/docs/entity-manager) explain it better. I wrote that thinking that it might be helpful during implementation, but I need to rewrite all of the repository mocking stuff in my tests